### PR TITLE
Chore/gh actions

### LINKED ISSUE #216: Move CI/CD to GH actions
Move our deployment pipeline to GH actions for easier maintenance and integration with github events e.g. release after required approvals, branch creating, tagging etc.

### PR DESCRIPTION
**Pull request by holygits**
closes #216 and #225

- Runs CI and release via github actoins
- Adds auto scheduled job to test upstream @polkadot packages for breaking changes

Other:
- Minor tidy ups to jest configs and docker-compose file
- Remove cz-lerna-changelog > @semantic-release/commit-analyzer as they're slow and unused

### CODE REVIEW

**Review by KarishmaBothara — APPROVED** [review]
lvgtm

File: .github/workflows/ci.yml
Comment (alexsednz) [inline .github/workflows/ci.yml:18]: Isn't this version very old?

File: .github/workflows/ci.yml
Comment (alexsednz) [inline .github/workflows/ci.yml:24]: Would yarn instructions generate this file?

Comment (holygits) [reply .github/workflows/ci.yml]: good catch!

**Comment by holygits** [comment]
will merge this after other outstanding PRs are merged first

**Pull request merged**
